test/SlowTests/Server/Documents/PeriodicExport/PeriodicExportTests.cs

```diff
@@ -54,31 +54,28 @@ public PeriodicExportTestsSlow()
                     .GetField(nameof(PeriodicExportRunner.MaxTimerTimeout), BindingFlags.Instance | BindingFlags.Public)
                     .SetValue(periodicExportRunner, TimeSpan.FromMilliseconds(5));
 
-                using (var commands = store.Commands())
+                var operation = new GetPeriodicBackupStatusOperation();
+                    //await store.Admin.Server.SendAsync(new ConfigurePeriodicBackupOperation(config, store.DefaultDatabase));
+                SpinWait.SpinUntil(() =>
```

Comment:
I haven't wrote those test i just modified them to work.
I don't like the use of SpinWait for polling...